DEL-34234: Credit-billing launch (Phase 1) — Billing/Plans on credits, remove monthly usage ceiling (#292)
speechmatics/docs: docs/administration/billing.mdx

---
title: Billing
description: Manage your payment card, understand how credits are used, and view and download invoices.
---

# Billing

> On the Billing page you add or manage your payment card, review your credits, and see what you have been charged. Free users add a card here to upgrade to Pro. Managing billing requires the Admin role.

Reviewer: Nit: ```suggestion
On the Billing page you add or manage your payment card, review your credits, and see your invoices. Free users add a card here to upgrade to Pro. Managing billing requires the Admin role.
```

Author: Done. Updated to 'see your invoices'

## Remove a payment card

:::warning

> Removing your payment card downgrades your plan to Free. Usage beyond the Free allowance stops being available.

Reviewer: We have to mention somewhere that if you remove the payment card and you don't have any credits you will be blocked from using our apis.

Author: Added a note in Remove a payment card covering this — worded it around credits running out rather than blocking, to keep it friendly.

Reviewer: Nice, I like it
